Tests/Dbal/Logging/BacktraceLoggerTest.php

```diff
@@ -9,6 +9,7 @@
 
 class BacktraceLoggerTest extends TestCase
 {
+    /** @group legacy */
```

Comment:
Given that the whole BacktraceLogger is a legacy thing (as it uses the legacy SQLLogger), I suggest putting that tag on the class level of the test (even though it is unlikely that we will introduce more tests for that legacy class)